pom.xml

```diff
@@ -15,7 +15,9 @@
 	<properties>
 		<spring-cloud-dataflow.version>1.0.0.BUILD-SNAPSHOT</spring-cloud-dataflow.version>
 		<spring-cloud-deployer-cloudfoundry.version>1.0.0.BUILD-SNAPSHOT</spring-cloud-deployer-cloudfoundry.version>
-		<reactor.version>2.5.0.M3</reactor.version>
+		<spring-cloud-sso-connector.version>1.1.0.RELEASE</spring-cloud-sso-connector.version>
+		<reactor.version>3.1.5.RELEASE</reactor.version>
+		<reactor-netty.version>0.7.5.RELEASE</reactor-netty.version>
```

Comment:
I did not see a need to specify reactor netty when looking into this, I hadn't seen your PR.

See changes in 3d699e3f9e6e2f10801770f49613766562cddb3b